e2e/src/pages/app.po.ts

```diff
@@ -69,6 +66,10 @@ export class AppPage {
     await browser.wait(until.presenceOf(element(by.css('div[class="page"'))), 15000, 'PDF viewer taking too long to load');
   }
 
+  async waitForImageToLoad() {
```

Comment:
Do we need this duplicate code ? waitforImageToLoad I assume you are waiting in the image viewer and that has the same class as pdf ? ( .page) ?

Reply:
Am sorry I forgot to remove that duplicate code

Reply:
deleted unused code

Reply:
great